### DISCUSSION TITLE
Multi-participant collaboration

### DISCUSSION THREAD

**Discussion by laultman** [StereoKit/StereoKit#944] [Q&A]
@maluoi Hi, Nick. I've been reading up on multi-player type applications. Unity seems to have several multi-player app extensions available. Have you encountered this and maybe how would you approach the problem. I'm not making a game, but I do have customers that want to have multiple participants in calls. Right now I am using Teams to facilitate the multiple users. Now I have a customer request that requires an offline (local network) solution.
Just starting to research the problem and thought I would reach out to you for any ideas you might have or know about.

**Comment by maluoi** [comment]
I've done some with raw sockets in the past. In C#, it's not too bad, and can be kinda fun to code up! But I've also seen people using a variety of different C# networking packages. I don't have any firsthand experiences with these packages in StereoKit, but given your preference for offline/local, most of the more complete and notable ones might be tricky to use anyhow. So I'm afraid I don't have any good recommendations for anything specific off the shelf 😅

**Comment by laultman** [comment]
Thanks, I'll let you know how I progress.
